Documentation/gitlab-ci.md

```diff
@@ -175,6 +175,105 @@ check_interval = 0  # Default poll interval
+    #OffPeakPeriods = ["* * 0-7,20-23 * * mon-fri *", "* * * * * sat,sun *"]
+    #OffPeakIdleTime = 300
+
+[[runners]]
+  name = "mender-qa-master"
+  limit = 40
+  output_limit = 512000
+  url = "https://gitlab.com/"
+  token = "<edited>"
+  executor = "docker+machine"
+  [runners.custom_build_dir]
+  [runners.cache]
+  [runners.docker]
+    tls_verify = false
+    image = "ubuntu:22.04"
+    privileged = true
+    disable_entrypoint_overwrite = true
+    oom_kill_disable = false
+    disable_cache = false
+    volumes = ["/dev/shm:/dev/shm", "/cache", "/dind/certs:/certs"]
+    shm_size = 8589934592
+  [runners.machine]
+    IdleCount = 0
+    IdleScaleFactor = 0.0
+    IdleCountMin = 0
+    IdleTime = 300
+    MachineDriver = "google"
+    MachineName = "gitlab-runner-slave-%s"
+    MachineOptions = ["google-project=mender-gitlab-runners", "google-machine-type=n1-highcpu-16", "google-disk-size=100", "google-machine-image=https://www.googleapis.com/compute/v1/projects/mender-gitlab-runners/global/images/nested-virt-ubuntu-2204-jammy-v20220506", "google-tags=mender-qa-slave", "google-zone=northamerica-northeast1-b", "google-use-internal-ip=true", "google-scopes=https://www.googleapis.com/auth/devstorage.read_write,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/cloud-platform"]
+    #OffPeakPeriods = ["* * 0-7,20-23 * * mon-fri *", "* * * * * sat,sun *"]
+    #OffPeakIdleTime = 300
+
+[[runners]]
+  name = "Ubuntu 22.04 Test Runner (highcpu)"
+  url = "https://gitlab.com/"
+  token = "<edited>"
+  executor = "docker+machine"
+  limit = 5
+  output_limit = 512000
+  [runners.custom_build_dir]
+  [runners.cache]
+  [runners.docker]
+    tls_verify = false
+    image = "ubuntu:22.04"
+    privileged = true
+    disable_entrypoint_overwrite = true
+    oom_kill_disable = false
+    disable_cache = false
+    volumes = ["/dev/shm:/dev/shm", "/cache", "/dind/certs:/certs"]
+    shm_size = 8589934592
+  [runners.machine]
+    IdleCount = 0
+    IdleScaleFactor = 0.0
+    IdleCountMin = 0
+    IdleTime = 300
+    MachineDriver = "google"
+    MachineName = "testmachine-%s"
+    MachineOptions = ["google-project=mender-gitlab-runners", "google-machine-type=n1-highcpu-16", "google-disk-size=100", "google-machine-image=https://www.googleapis.com/compute/v1/projects/mender-gitlab-runners/global/images/nested-virt-ubuntu-2204-jammy-v20220506", "google-tags=mender-qa-slave", "google-zone=northamerica-northeast1-b", "google-use-internal-ip=true", "google-scopes=https://www.googleapis.com/auth/devstorage.read_write,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/cloud-platform"]
+    #OffPeakPeriods = ["* * 0-7,20-23 * * mon-fri *", "* * * * * sat,sun *"]
+    #OffPeakIdleTime = 300
```

Comment:
@alfrunes I believe this third one is not in use. I cannot see any machine with prefix `testmachine-...` in GCP. Do you remember why you created it? Can it be removed?

Reply:
This runner configuration is indeed not being used anymore, it was used in the transition phase to ubuntu 22.04. I wouldn't remove the configuration before we've removed the corresponding tags from gitlab though.

Reply:
Discussed f2f. I will comment out the configuration from the VM and check again on Monday before deleting it completely :+1:

Reply:
Confirmed, it is the one we suspected

![image](https://user-images.githubusercontent.com/3168644/194818102-39fa9297-a450-497f-8150-c7dc226443e1.png)

Cleaning it up now as a follow-up from this PR.